include/odp/api/spec/queue_types.h

```diff
@@ -130,14 +131,16 @@ typedef struct odp_queue_capability_t {
 	  * types are used simultaneously. */
 	uint32_t max_queues;
 
-	/** Maximum number of ordered locks per queue */
-	uint32_t max_ordered_locks;
+	/** @deprecated Use max_ordered_locks field of
+	 * odp_schedule_capability_t instead */
+	uint32_t ODP_DEPRECATE(max_ordered_locks);
```

Comment:
This commit: "api: queue, schedule: move scheduler capabilities to scheduler" (dbf52ac22f09c7dc3987b89059255ffcdc323ea8) cannot move yet, due to deprecation of queue capa.